metadata.go

```diff
@@ -192,6 +200,86 @@ func GetImageToMDMap(imageMDs []ImageMetadataInfo) (imageToMDMap map[string][]Im
 	return
 }
 
+// getLocalImages queries the local Docker daemon for list of images.
+func getLocalImages() (imageMap map[ImageIDType][]RepoTagType, e error) {
+
+	// query a list of images from Docker daemon
+	response, e := listImages()
+	if e != nil {
+		return nil, e
+	}
+	blog.Info(string(response))
+	// parse JSON
+	var localImageList []LocalImageStruct
+	if e = json.Unmarshal(response, &localImageList); e != nil {
+		return nil, e
+	}
+
+	// make map from each imageID to all of its aliases (repo+tag)
+	imageMap = make(map[ImageIDType][]RepoTagType)
+	for _, localImage := range localImageList {
+		imageID := ImageIDType(localImage.ID)
+		for _, repoTag := range localImage.RepoTags {
+			// skip images with no repo:tag
+			if repoTag == "" {
```

Comment:
I'm seeing some images with no repo:tag showing up in my output.
                        "Image": "a5573d9ea98b4b30efa312502b4b82a3d347827966b5c73dcaea5dba04a7faf0",
                        "Datetime": "2015-07-12T02:24:10.314060361Z",
                        "Repo": "\u003cnone\u003e",
                        "Tag": "\u003cnone\u003e",
                        "Size": 553,
                        "Author": "",
                        "Checksum": "",
                        "Comment": "",
                        "Parent": "6a3f62ed92a63a07265c0658cf8fb4768c2767581c0667ba5f90e7027457a1d9"

The API response shows the RepoTags as follows:
"RepoTags":["\u003cnone\u003e:\u003cnone\u003e"]